app/services/account_creation/device_profile/verification_request.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module AccountCreation
+  module DeviceProfile
+    class VerificationRequest < Proofing::LexisNexis::Ddp::VerificationRequest
+      private
+
+      def build_request_body
+        {
+          api_key: config.api_key,
+          org_id: config.org_id,
+          account_email: applicant[:email],
+          event_type: 'ACCOUNT_CREATION',
+          policy: IdentityConfig.store.authentication_tmx_policy, # Need a sep policy from proofing
```

Comment:
Do we need to start passing `policy` for the existing proofing calls as well?

Reply:
I think since the verification request is separate. We dont need to pass in policy. as part of the config, but if we parametrize we could definitely